examples/index.md

```diff
@@ -877,15 +877,61 @@ __Available for the following carriers:__
 
 {% include examples/declared_value_dhl.md %}
 
+## Label format
+Some carriers are providing the shipping labels in different sizes and formats (have a look at our
+chart of the [carrier specific label sizes]({{ site.baseurl }}/concepts/#carrier-specific-label-sizes)
+we support for detailed information).
+
+You can configure a standard that you want us to use when creating shipping labels for each and
+every carrier from within the shipcloud backoffice, so you won't have to specify the format with
+every single request to our api. You can however define the format of the shipping label on a per
+shipment basis. So, when you're creating a shipping label via our api you can send us the size the
```

Comment:
Sollten wir hier nicht besser schreiben "you can send us the format" statt "size"?